src/assemble_workflow/fpm_builder.py

```diff
@@ -0,0 +1,94 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import os
+import shutil
+import subprocess
+from assemble_workflow.bundle_recorder import BundleRecorder
+
+
+class FpmBuilder:
+    def build(self, bundle_recorder: BundleRecorder, archive_path: str, distribution: str) -> None:
+        manifest_data = bundle_recorder.bundle_manifest.data["build"]
+        product_name = product_name_alt = manifest_data["name"].lower()
+        if product_name == "opensearch dashboards":
```

Comment:
The `if` doesn't seem necessary, you can do this for every product, it will leave `OpenSearch` alone.

Reply:
Test and work, thanks very much.